.github/workflows/coverage.yml

```diff
@@ -1,16 +1,18 @@
-name: Coverage
+name: CI
 
 on:
   pull_request:
     paths:
-    - '**.rs'
+      - '**.rs'
   push:
+    branches:
+      - main
     paths:
-    - '**.rs'
+      - '**.rs'
 
 jobs:
 
-  codecov:
+  coverage:
```

Comment:
just checking: this was included because it was convenient to fix now, right?

Reply:
Yes correct